From 764092fddb160ef0f03996ab45a8f4c246c34e1c Mon Sep 17 00:00:00 2001
From: Florian Fainelli <florian@openwrt.org>
Date: Mon, 31 Dec 2012 13:52:46 +0100
Subject: [PATCH 2/8] Config.in: add ARCH_HAS_UCONTEXT configuration symbol

This configuration symbol can be selected by an architecture supporting
the {get,set,swap,make}context user context manipulation functions.

Signed-off-by: Florian Fainelli <florian@openwrt.org>
---
 extra/Configs/Config.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/extra/Configs/Config.in b/extra/Configs/Config.in
index 4bb6812..d0204bb 100644
--- a/extra/Configs/Config.in
+++ b/extra/Configs/Config.in
@@ -261,6 +261,9 @@ config ARCH_HAS_NO_LDSO
 	bool
 	select ARCH_HAS_NO_SHARED
 
+config ARCH_HAS_UCONTEXT
+	bool
+
 config HAVE_SHARED
 	bool "Enable shared libraries"
 	depends on !ARCH_HAS_NO_SHARED
-- 
1.8.1.2

